### PR TITLE
Mentioned Synapol submarines use in its description

### PR DESCRIPTION
**Pull request by OcelotWalrus**

### CODE REVIEW

File: mods/hv/languages/rules/en.ftl [hunk 1]
```diff
@@ -49,6 +49,7 @@ faction-synapol =
 
     Faction Variations:
         - Uses helicopters as air units
+        - Uses submarines as advanced naval units
```
Comment (Mailaender) [inline mods/hv/languages/rules/en.ftl:52]: This is unbalanced as there is no counterpart statement on the other faction.

Comment (OcelotWalrus) [reply mods/hv/languages/rules/en.ftl:52]: What do you want me to say? There's no counterpart, I think.

Comment (OcelotWalrus) [reply mods/hv/languages/rules/en.ftl:52]: If not saying "Uses boats as advanced naval units" in Yuruki's description.

Comment (Mailaender) [reply mods/hv/languages/rules/en.ftl:52]: Yeah that is lame.

**Pull request merged**